### ISSUE TITLE
Netdata nightly builds not updating automatically

### ISSUE THREAD

**Issue by TheOldMan2000** [netdata/netdata#9763]
Using Nightly builds, 64bit Netdata. Automated updates stopped working again for last few weeks. If I run the updater or one line static installer (with or without --nightly-builds option) it is disabled forcing manual updates.

Other issues I'm having, in case somehow related:

- SystemD charts section enabled in Netdata config but not displayed for some time, several weeks or so.
- Chart height adjusters (Up/down arrow icons) don't work on iPad.
- All Charts just suddenly disappear (open bug on this [here])(https://github.com/netdata/netdata/issues/9745)

##### Bug report summary

```

 --- Install (but not enable) netdata updater tool --- 
Update script is located at /opt/netdata/usr/libexec/netdata/netdata-updater.sh

 --- Check if we must enable/disable the netdata updater tool --- 
You chose *NOT* to enable auto-update, removing any links to the updater from cron (it may have happened if you are reinstalling)

Did not find any cron entries to remove
```

I've NEVER chosen NOT to enable auto-update, so this isn't correct and may be part of the issue.

##### OS / Environment

CENTOS 7.8 virtuozzo [vpsXXXXXX]  WHM/CPANEL v88.0.13

```

uname -a; grep -Hv "^#" /etc/*release
Linux vpsXXXXXX.inmotionhosting.com 3.10.0-1062.12.1.vz7.131.10 #1 SMP Mon Mar 16 15:39:59 MSK 2020 x86_64 x86_64 x86_64 GNU/Linux
/etc/centos-release:CentOS Linux release 7.8.2003 (Core)
/etc/os-release:NAME="CentOS Linux"
/etc/os-release:VERSION="7 (Core)"
/etc/os-release:ID="centos"
/etc/os-release:ID_LIKE="rhel fedora"
/etc/os-release:VERSION_ID="7"
/etc/os-release:PRETTY_NAME="CentOS Linux 7 (Core)"
/etc/os-release:ANSI_COLOR="0;31"
/etc/os-release:CPE_NAME="cpe:/o:centos:centos:7"
/etc/os-release:HOME_URL="https://www.centos.org/"
/etc/os-release:BUG_REPORT_URL="https://bugs.centos.org/"
/etc/os-release:
/etc/os-release:CENTOS_MANTISBT_PROJECT="CentOS-7"
/etc/os-release:CENTOS_MANTISBT_PROJECT_VERSION="7"
/etc/os-release:REDHAT_SUPPORT_PRODUCT="centos"
/etc/os-release:REDHAT_SUPPORT_PRODUCT_VERSION="7"
/etc/os-release:
/etc/redhat-release:CentOS Linux release 7.8.2003 (Core)
/etc/system-release:CentOS Linux release 7.8.2003 (Core)

```

```
/opt/netdata/usr/libexec/netdata/netdata-updater.sh --help
Entering /tmp/netdata-updater-CuD6VO

  ^
  |.-.   .-.   .-.   .-.   .  netdata
  |   '-'   '-'   '-'   '-'   real-time performance monitoring, done right!
  +----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+--->

  (C) Copyright 2017, Costa Tsaousis
  All rights reserved
  Released under GPL v3+

  You are about to install netdata to this system.
  netdata will be installed at:

                    /opt/netdata

  The following changes will be made to your system:

  # USERS / GROUPS
  User 'netdata' and group 'netdata' will be added, if not present.

  # LOGROTATE
  This file will be installed if logrotate is present.

   - /etc/logrotate.d/netdata

  # SYSTEM INIT
  This file will be installed if this system runs with systemd:

   - /lib/systemd/system/netdata.service

   or, for older Centos, Debian/Ubuntu or OpenRC Gentoo:

   - /etc/init.d/netdata         will be created


  This package can also update a netdata installation that has been
  created with another version of it.

  Your netdata configuration will be retained.
  After installation, netdata will be (re-)started.

  netdata re-distributes a lot of open source software components.
  Check its full license at:
  https://github.com/netdata/netdata/blob/master/LICENSE.md
Creating directory /opt/netdata
Verifying archive integrity...  100%   All good.
Uncompressing netdata, the real-time performance and health monitoring system  100%  
 --- Attempt to create user/group netdata/netadata --- 
Group 'netdata' already exists.
User 'netdata' already exists.
 --- Add user netdata to required user groups --- 
Group 'docker' does not exist.
 FAILED  Failed to add netdata user to secondary groups 

Group 'nginx' does not exist.
 FAILED  Failed to add netdata user to secondary groups 

Group 'varnish' does not exist.
 FAILED  Failed to add netdata user to secondary groups 

Group 'haproxy' does not exist.
 FAILED  Failed to add netdata user to secondary groups 

User 'netdata' is already in group 'adm'.
User 'netdata' is already in group 'nsd'.
Group 'proxy' does not exist.
 FAILED  Failed to add netdata user to secondary groups 

Group 'squid' does not exist.
 FAILED  Failed to add netdata user to secondary groups 

Group 'ceph' does not exist.
 FAILED  Failed to add netdata user to secondary groups 

User 'netdata' is already in group 'nobody'.
Group 'I2C' does not exist.
 FAILED  Failed to add netdata user to secondary groups 

 --- Check SSL certificates paths --- 
 --- Install logrotate configuration for netdata --- 
[/opt/netdata]# chmod 644 /etc/logrotate.d/netdata 
 OK   

 --- Telemetry configuration --- 
You can opt out from anonymous statistics via the --disable-telemetry option, or by creating an empty file /opt/netdata/etc/netdata/.opt-out-from-anonymous-statistics 

 --- Install netdata at system init --- 
Installing systemd service...
[/opt/netdata]# cp system/netdata.service /lib/systemd/system/netdata.service 
 OK   

[/opt/netdata]# systemctl daemon-reload 
 OK   

[/opt/netdata]# systemctl enable netdata 
 OK   

 --- Install (but not enable) netdata updater tool --- 
Update script is located at /opt/netdata/usr/libexec/netdata/netdata-updater.sh

 --- Check if we must enable/disable the netdata updater tool --- 
You chose *NOT* to enable auto-update, removing any links to the updater from cron (it may have happened if you are reinstalling)

Did not find any cron entries to remove

 --- creating quick links --- 
[/opt/netdata]# rm sbin 
 OK   

[/opt/netdata]# ln -s bin sbin 
 OK   

[/opt/netdata/usr]# rm bin 
 OK   

[/opt/netdata/usr]# ln -s ../bin bin 
 OK   

[/opt/netdata/usr]# rm sbin 
 OK   

[/opt/netdata/usr]# ln -s ../bin sbin 
 OK   

[/opt/netdata/usr]# rm local 
 OK   

[/opt/netdata/usr]# ln -s . local 
 OK   

[/opt/netdata]# rm netdata-configs 
 OK   

[/opt/netdata]# ln -s etc/netdata netdata-configs 
 OK   

[/opt/netdata]# rm netdata-web-files 
 OK   

[/opt/netdata]# ln -s usr/share/netdata/web netdata-web-files 
 OK   

[/opt/netdata]# rm netdata-plugins 
 OK   

[/opt/netdata]# ln -s usr/libexec/netdata netdata-plugins 
 OK   

[/opt/netdata]# rm netdata-dbs 
 OK   

[/opt/netdata]# ln -s var/lib/netdata netdata-dbs 
 OK   

[/opt/netdata]# rm netdata-metrics 
 OK   

[/opt/netdata]# ln -s var/cache/netdata netdata-metrics 
 OK   

[/opt/netdata]# rm netdata-logs 
 OK   

[/opt/netdata]# ln -s var/log/netdata netdata-logs 
 OK   

[/opt/netdata/etc/netdata]# rm orig 
 OK   

[/opt/netdata/etc/netdata]# ln -s ../../usr/lib/netdata/conf.d orig 
 OK   

 --- fix permissions --- 
[/opt/netdata]# chmod g+rx\,o+rx /opt 
 OK   

[/opt/netdata]# chown -R netdata:netdata /opt/netdata 
 OK   

 --- fix plugin permissions --- 
[/opt/netdata]# chown root:netdata usr/libexec/netdata/plugins.d/apps.plugin 
 OK   

[/opt/netdata]# chmod 4750 usr/libexec/netdata/plugins.d/apps.plugin 
 OK   

[/opt/netdata]# chown root:netdata usr/libexec/netdata/plugins.d/ioping 
 OK   

[/opt/netdata]# chmod 4750 usr/libexec/netdata/plugins.d/ioping 
 OK   

[/opt/netdata]# chown root:netdata usr/libexec/netdata/plugins.d/cgroup-network 
 OK   

[/opt/netdata]# chmod 4750 usr/libexec/netdata/plugins.d/cgroup-network 
 OK   

[/opt/netdata]# chown root:netdata bin/fping 
 OK   

[/opt/netdata]# chmod 4750 bin/fping 
 OK   

Configure TLS certificate paths
Using /etc/pki/tls for TLS configuration and certificates
Save install options
 --- starting netdata --- 
 --- Restarting netdata instance --- 

Stopping all netdata threads
[/opt/netdata]# stop_all_netdata 
 OK   

Starting netdata using command 'systemctl start netdata'
[/opt/netdata]# systemctl start netdata 
 OK   


  ^
  |.-.   .-.   .-.   .-.   .-.   .  netdata              .-.   .-.   .-.   .-
  |   '-'   '-'   '-'   '-'   '-'   is installed now!  -'   '-'   '-'   '-'  
  +----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+--->

[/opt/netdata]# chmod 0644 /opt/netdata/etc/netdata/netdata.conf 
 OK   

Switching back to /root

```

##### Netdata version

netdata v1.24.0-49-gc6ba56af

 (plus previous releases in last 2-3 weeks approx)

##### Steps To Reproduce

/opt/netdata/usr/libexec/netdata/netdata-updater.sh

##### Expected behavior

Netdata to detect auto updater is broken and attempt to fix itself if a configuration issue or bug preventing this from working to be squished and normal auto updates restored.

P.S. One daily new version check is too slow, why not one check say every 6 hours or once per hour?

**Comment by Ferroin** [comment]
Could you please share the contents of `/opt/netdata/etc/netdata/.environment from the system you're seeing this issue on?

**Comment by TheOldMan2000** [comment]
Yes, no problem:

```
# Created by installer
PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
CFLAGS="-static -O3 -I/openssl-static/include"
LDFLAGS="-static -L/openssl-static/lib"
NETDATA_PREFIX="/opt/netdata"
NETDATA_CONFIGURE_OPTIONS=" --enable-cloud"
NETDATA_ADDED_TO_GROUPS=" adm squid nobody"
INSTALL_UID="0"
NETDATA_GROUP="netdata"
REINSTALL_OPTIONS=""
RELEASE_CHANNEL="nightly"
IS_NETDATA_STATIC_BINARY="yes"
NETDATA_LIB_DIR="/opt/netdata/var/lib/netdata"

 ```

**Comment by Ferroin** [comment]
Sorry about the delay in responding.

It looks like this is probably a duplicate of #8782 and will thus probably be fixed by #8808

**Comment by TheOldMan2000** [comment]
Thank you, much appreciated.
